core/shared/utils/bh_log.c

```diff
@@ -2,6 +2,9 @@
  * Copyright (C) 2019 Intel Corporation.  All rights reserved.
  * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
  */
+#ifdef linux
+#include <stdio.h>
+#endif
```

Comment:
Had better not include <stdio.h> in non-platform source files, `#include "bh_platform.h"` instead, which includes platform_internal.h, and the latter includes `<stdio.h>`.